cmd/bnsd/app/testdata/fixtures/app.go

```diff
@@ -81,6 +82,26 @@ func appStateGenesis(keyAddress weave.Address) []byte {
 				CollectorAddress: weave.Condition("dist/revenue/0000000000000001").Address(),
 				MinimalFee:       coin.NewCoin(0, 10000000, "FRNK"),
 			},
+			"migration": migration.Configuration{
+				Admin: weave.Condition("multisig/usage/0000000000000001").Address(),
+			},
+		},
+		"initialize_schema": []string{
```

Comment:
This doesn't take a schema number? What if we launch with schema 3 for example

Reply:
It only allows for initialization with schema 1. I was thinking about it and I am decided that there is no need for initialization with a higher number.
My thinking was that genesis is loaded only once during the node initialization. This value is to only enable message/model. All further changes should be made via message.

The only case where having a higher initialization value is required is when we use higher than 1 schema before the release. I think we don't have to do this.

What do you think?

Reply:
> The only case where having a higher initialization value is required is when we use higher than 1 schema before the release. I think we don't have to do this.

Not now, but if we do some upgrades on bnsd (eg. gov to schema 3) and we want to launch a new testnet to demo this. Why not start at 3? Why have to manually upgrade every time, rather than start directly where you want to be?

Reply:
Continued in https://github.com/iov-one/weave/pull/556#discussion_r278925793